tests/XdebugHandlerTest.php

```diff
@@ -121,7 +122,7 @@ public function testEnvAllowWithEmptyScanDir()
         putenv('PHP_INI_SCAN_DIR');
         $xdebug = new XdebugHandlerMock($loaded);
         $xdebug->check();
-        $this->assertEquals('', getenv('PHP_INI_SCAN_DIR'));
+        $this->assertEmpty(getenv('PHP_INI_SCAN_DIR'));
```

Comment:
This, and the other `assertEquals('', ...)` tests are testing for a string.

Reply:
Gonna rollback this changes soo

Reply:
Thanks. If you use `assertSame`, rather than just revert the changes then you will fix a bug.